CMakeLists.txt

```diff
@@ -47,6 +47,8 @@ lxqt_translate_ts(QM_FILES
     ${UIS}
     INSTALL_DIR
     "${LXQT_TRANSLATIONS_DIR}/${PROJECT_NAME}"
+    PULL_TRANSLATIONS ${PULL_TRANSLATIONS}
```

Comment:
Yes, forgot that... but as there is no way to get all the possible (nested) options from all used modules in `cmake`, it may be good to pull them into the top CMakeLists.txt file.